src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -28,20 +27,17 @@
 class DelegatingLoader extends BaseDelegatingLoader
 {
     protected $parser;
-    protected $logger;
     private $loading = false;
 
     /**
      * Constructor.
      *
      * @param ControllerNameParser    $parser   A ControllerNameParser instance
-     * @param LoggerInterface         $logger   A LoggerInterface instance
      * @param LoaderResolverInterface $resolver A LoaderResolverInterface instance
      */
-    public function __construct(ControllerNameParser $parser, LoggerInterface $logger = null, LoaderResolverInterface $resolver)
+    public function __construct(ControllerNameParser $parser, LoaderResolverInterface $resolver)
```

Comment:
The deprecation warning should be triggered when a logger is passed as the second argument in Symfony 2.8. You can take a look at the [FragmentHandler](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/Fragment/FragmentHandler.php#L53-L62) for an example.